sdk/ml/azure-ai-ml/azure/ai/ml/_internal/entities/node.py

```diff
@@ -186,6 +187,15 @@ def __init__(self, **kwargs):
         self._hdinsight_spark_job_name: str = kwargs.pop("hdinsight_spark_job_name", None)
         self._init = False
 
+    @property
+    def compute_name(self) -> str:
+        """Name of the compute to be used."""
+        return self._compute_name
+
+    @compute_name.setter
+    def compute_name(self, value: str):
+        self._compute_name = value
```

Comment:
will binding supported on this?

Reply:
Should be supported by default. Let me add a test for this